evaluation/metrics/train_file_metrics.py

```diff
@@ -73,6 +73,15 @@ def dataperf(train, test, constructor_args):
 
     preds_formatted: [{"uid": test_id_0, "labels": ["Cat", "Dog", ...]}, ...]
     """
+
+    # For Dataperf, we want to cap the number of train examples to 1000 for each
+    # dataset.
+    total_number_of_train_examples = 0
+    for _, value in train.items():
+        for _, _ in value.items():
+            total_number_of_train_examples += 1
+            assert total_number_of_train_examples <= 1000
```

Comment:
Yes. That whole thing is run inside a try/catch block, and it will send an email to the user if the train file didn't work. I think I should clean it up in the future

Reply:
ah ok makes sense